### PR TITLE
health-Book-doc backend - V.1.0

### PR DESCRIPTION
**Pull request by ankitt26**
Hello Code Reviewer, Hope you're doing good 👋🏿

### In this Project we did-
  - Set up Rails backend for the application
  - Set up `Api` only rails
  - Create Models
  - Set up `devise - jwt `for authentication
  - Use the JWT token to verify user
  - Set up Authorization using `cancancan`.
 -  Set up controllers for Doctors, appointments and users
 - Add validations 
 - Create relations between models
 -  Use `Postgres` as your database 
 -  Write Tests using `Rspec`
 - Write `API` Documentation
 - Deploy the backend on Render

> Link to 🎋 Doctors-Appoinments-Front-End pull request  [click here](https://github.com/adarachel/doctors-appointment-frontend/pull/16)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+```
```
Comment (Meltrust) [inline README.md:111]: - [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
#### Hi @ankitt26,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause





Cheers and Happy coding!👏👏👏


------

File: README.md (hunk 1 above)
Comment (ankitt26) [reply README.md:111]: Thanks for reviewing the Pull Request, I will make the required changes🙂.

**Review by Meltrust — APPROVED** [review]
#### Hi @ankitt26,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause





Cheers and Happy coding!👏👏👏


------

**Pull request merged**